### PR TITLE
[WIP] Move git content from FAQ to Git Troubleshoot

### LINKED ISSUE #191: Move Automation info out of FAQ and into Workspace Settings
1. Verify content in this section: 

https://meta.stoplight.io/docs/platform/ZG9jOjM5OTEzOA-faq-and-troubleshooting#cant-find-github-organization

And move here:
https://meta.stoplight.io/docs/platform/ZG9jOjM1NzUwMjEw-troubleshoot-git

2. Compare the content here:

https://meta.stoplight.io/docs/platform/ZG9jOjM5OTEzOA-faq-and-troubleshooting#why-are-files-missing-from-my-project-documentation

With the content here:
https://meta.stoplight.io/docs/platform/ZG9jOjM1NzUwMjEw-troubleshoot-git#troubleshoot-webhooks

And see if we can move the content from the FAQ to the Troubleshooting topic so it's in one place.

### PR DESCRIPTION
**Pull request by heitortsergent**
Fixes #191.

I've moved one question from the FAQ related to Git to our Troubleshoot Git page. Also moved the information about adding Stoplight's IP address from those pages to the main Git overview page, to help the user be aware of it without having to search for it in other pages.

I've also moved the FAQ and Troubleshooting page to the bottom of the navigation, I think it makes more sense to have it as the last page in our docs.

### CODE REVIEW

File: 2.-workspaces/configure-git/f-troubleshoot-git.md [hunk 1]
```diff
@@ -76,6 +76,14 @@ You can reinstall webhooks in Stoplight if you are unable to edit a Git project.
 
 Reinstall the webhooks on the **Automation** tab for the project that you are having issues with.
 
+## Can't find GitHub Organization 
+
+This could be due to missing permissions in GitHub. Navigate to https://github.com/settings/applications and find the Stoplight app. Click on it to go into the settings.
```
Comment (pamgoodrich) [inline 2.-workspaces/configure-git/f-troubleshoot-git.md:81]: Change "Click" to "Select". Select is more inclusive. https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click

File: c.troubleshooting.md [hunk 2]
```diff
@@ -43,17 +43,15 @@ The `toc.json` file can be used to restrict what is published in the
 documentation view for a project. This is useful for controlling the
 presentation of the sidebar.
 
-If you have a `toc.json` file present in your project, double check the contents
+If you have a `toc.json` file present in your project, double-check the contents
```
Comment (pamgoodrich) [inline c.troubleshooting.md:46]: Maybe change "double-check" to "verify"?

File: c.troubleshooting.md [hunk 3]
```diff
@@ -43,17 +43,15 @@ The `toc.json` file can be used to restrict what is published in the
...
 
 For more information on this feature, see
 [here](4.-documentation/Sidebar/d.table-of-contents.md).
 
 ### Is the branch you are on published in the project settings?
 
-Verify that the branch you are pushing your updates to is published and
-listed. The published and available branches can be found under the project
-**Settings** page.
+Verify that the branch you are pushing your updates to is published and listed. The published and available branches can be found under the project **Settings** page.
```
Comment (pamgoodrich) [inline c.troubleshooting.md:54]: I might not hurt to link to this topic: https://meta.stoplight.io/docs/platform/ZG9jOjQ2NDU5Mw-branch-management

**Review by pamgoodrich — APPROVED** [review]
A few small changes, and then you are good to merge.

File: c.troubleshooting.md (hunk 3 above)
Comment (heitortsergent) [reply c.troubleshooting.md:54]: That topic is linked right after the image under this paragraph, so I'll just leave that as is. 😃

**Pull request merged**